include/grl/kuka/KukaDriver.hpp

```diff
@@ -261,6 +261,49 @@ namespace grl { namespace robot { namespace arm {
         }
    }
 
+  bool setPositionControlMode()
```

Comment:
Can each of these be a simple set() function which takes a CartesianImpedenceControlMode object, a JointImpedenceControlMode, or a to be created PositionControlMode object?

My description here might need a little more thinking...